src/Hazelcast.Net.Examples/Properties/launchSettings.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
not sure we want to add that file?

Reply:
It was a mistake. Fixed at https://github.com/hazelcast/hazelcast-csharp-client/pull/719/commits/6dcef8a55d73060af97c494821575516c8cd240e